docs/DEVELOPING.md

```diff
@@ -640,7 +640,6 @@ Microsoft SQL Server versions 2017, 2019, and Azure SQL, Amazon RDS for Microsof
 
 **Prerequisites**
 
-- The Microsoft SQL connector requires an ODBC driver to execute. Connector supports `Microsoft ODBC Driver 18`. To install ODBC driver please refer following [documentation](https://learn.microsoft.com/en-us/sql/connect/odbc/linux-mac/installing-the-driver-manager).
```

Comment:
do we also need changes to the configuration docs? See line 710: `##### mssql_driver` (too far below the diff for github to let me comment on it)